.setup/update_system.sh

```diff
@@ -29,6 +29,9 @@ apt-get install libzbar0 --yes
 
 ##source ${CURRENT_DIR}/distro_setup/setup_distro.sh
 
+# update packages
+apt-get update && apt-get upgrade
```

Comment:
I think you would want to put a `-y` after the apt-get upgrade incase it asks you for a prompt to install any upgradeable packages